programmierkurs_fpcu.tex

```diff
@@ -75,7 +75,7 @@
   keywordstyle=\color{blue},       % keyword style
   language=C,                      % the language of the code
   morekeywords={*,...},            % if you want to add more keywords to the set
-  numbers=none,                    % where to put the line-numbers; possible values are (none, left, right)
+  numbers=left,                    % where to put the line-numbers; possible values are (none, left, right)
```

Comment:
Ich wollte die nur in den listings zeigen, in denen sie auch benutzt werden. Das waren nicht so viele, wenn ich das richtig gesehen habe.

Reply:
Okay, können wir gerne so machen. Habe die entprechenden Stellen in meinen Notizen markiert, kann ich also noch ändern.